knexfile.js

```diff
@@ -1,8 +1,10 @@
 // Update with your config settings.
+const pg = require('pg');
+
+pg.defaults.ssl = true;
```

Comment:
What's the reason for activating ssl connections ?

Reply:
The ssl configuration is only necessary if you are using a database on Heroku or another provider that requires an SSL connection. dotenv allows us to retrieve environmental variables from a .env file! That variable is a standard PostgreSQL connection string: